### PR TITLE
Modify AddCommand AddCommandParser to TagCommand and TagCommandParser

### LINKED ISSUE #48: Modify AddCommand AddCommandParser to TagCommand and TagCommandParser

### PR DESCRIPTION
**Pull request by luo-git**
Also switches to the new UI.

Resolves #48

### CODE REVIEW

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public void execute_storageThrowsIoException_throwsCommandException() {
         logic = new LogicManager(model, storage);
 
         // Execute add command
-        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY
+        String addCommand = TagCommand.COMMAND_WORD + NAME_DESC_AMY
                 + FILE_ADDRESS_DESC_AMY;
```
Comment (Ma-Yueran) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:81]: Seems that this test also need to be changed in the future. But I think maybe this does not matter now.

**Review by Ma-Yueran — APPROVED** [review]
Looks good to me.

File: src/test/java/seedu/address/logic/commands/TagCommandIntegrationTest.java [hunk 2]
```diff
@@ -32,14 +32,14 @@ public void execute_newTag_success() {
...
-                String.format(AddCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
+        assertCommandSuccess(new TagCommand(validPerson), model,
+                String.format(TagCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
     }
 
     @Test
     public void execute_duplicateTag_throwsCommandException() {
         Tag personInList = model.getAddressBook().getTagList().get(0);
-        assertCommandFailure(new AddCommand(personInList), model, AddCommand.MESSAGE_DUPLICATE_TAG);
+        assertCommandFailure(new TagCommand(personInList), model, TagCommand.MESSAGE_DUPLICATE_TAG);
```
Comment (li-s) [inline src/test/java/seedu/address/logic/commands/TagCommandIntegrationTest.java]: Shouldnt personInList be tagInList instead?

**Review by li-s — CHANGES_REQUESTED** [review]
Almost LGTM

File: src/test/java/seedu/address/logic/commands/TagCommandIntegrationTest.java (hunk 2 above)
Comment (luo-git) [reply src/test/java/seedu/address/logic/commands/TagCommandIntegrationTest.java]: Roger that

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 1 above)
Comment (luo-git) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:81]: We need to change all the entities in testUtil.

**Review by li-s — APPROVED** [review]

**Pull request merged**